### DISCUSSION TITLE
Doesn't compile on ESP-IDF

### DISCUSSION THREAD

**Discussion by DerTiger1** [omerfaruk-aran/esphome_samsung_hvac_bus#241] [General]
Again an issue with isp-idf on the current version:
![grafik](https://github.com/user-attachments/assets/294886fc-199a-45f6-ab5d-10eb801e1d19)

**Comment by omerfaruk-aran** [comment]
Hi @DerTiger1,

The issue you reported regarding the ISP-IDF version compatibility and the esp_mqtt_client_config_t structure has been addressed. The fields like host, port, username, and password have been updated to align with the latest ESP-IDF changes.

Please pull the latest changes from the repository and try building your project again. If the problem persists, feel free to share additional logs or details.

Thanks for your feedback,

Best regards,
